portfolio/src/main/webapp/style.css

```diff
@@ -33,4 +33,63 @@ h1 {
 
 .nav-links a:hover {
   background-color: lightgrey;
+}
+
+#image-links a {
+  float: none;
+  text-align: center;
+  padding-right: 20px;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/*image gallery styling */
+
+/* grid: Four equal columns that floats next to each other */
+.column {
+  float: left;
+  width: 25%;
+}
+
+/* Style the images inside the grid */
```

Comment:
you can nest those two fields in to .column for a easier reading

Reply:
Sorry, which fields are you referring to?